index.d.ts

```diff
@@ -410,6 +429,12 @@ declare module '@xmldom/xmldom' {
 		 * [MDN Reference](https://developer.mozilla.org/docs/Web/API/Node/parentNode)
 		 */
 		readonly parentNode: Node | null;
+		/**
+		 * Returns the parent element.
```

Comment:
I would like to make it more explicit that it is just checking the direct parent, e.g.

> Returns the parent `Node` if it is of type `Element`, otherwise `null`.